src/main/scala/com/gu/support/workers/model/JsonWrapper.scala

```diff
@@ -4,4 +4,4 @@ package com.gu.support.workers.model
  * AWS Step Functions expect to be passed valid Json, as we want to encrypt the whole of the
  * state, we need to wrap it in a Json 'wrapper' object as a Base64 encoded String
  */
-case class JsonWrapper(state: String, error: Option[ExecutionError], encrypted: Boolean)
\ No newline at end of file
+case class JsonWrapper(state: String, error: Option[ExecutionError], requestInformation: RequestInfo)
```

Comment:
You might need to think about what will happen during deployment?  I'll discuss in person.

Reply:
But you've already thought of that  👍